extensions/amp-ad/0.1/amp-ad-ui.js

```diff
@@ -21,10 +21,7 @@ import {UX_EXPERIMENT} from '../../../src/layout';
 const TAG = 'AmpAdUIHandler';
 
 /** @const */
-const HOLDER_HTML =
-    `<div class='-amp-ad-default-holder'>
-    <div class='-amp-ad-tag'>Ad</div>
-    </div>`;
+const HOLDER_HTML = `<div class='-amp-ad-default-holder'></div>`;
```

Comment:
Please redo as a direct DOM construction. `innerHTML` is not allowed for cases like this.